pkg/config/virtio_test.go

```diff
@@ -16,23 +20,27 @@ type virtioDevTest struct {
 	errorMsg         string
 }
 
+var testImagePath = filepath.Join(os.TempDir(), "vfkit_test_disk.img")
```

Comment:
golang testing package also provides  https://pkg.go.dev/testing#T.TempDir , which has the benefit of being cleaned up automatically. But I can’t think of an easy way of using it given the way our tests are implemented, so your approach is fine.

Reply:
The tricky part is that we are using a table to define the tests (`virtioDevTests`), and some parts of the table need to know the filename: https://github.com/crc-org/vfkit/blob/16d5ddd714db62cf1bf21a2cdc1431021dd9a808/pkg/config/virtio_test.go#L31

But it looks like we can move the table (`virtioDevTests`) to `TestVirtioDevices` where we can define `testImagePath` locally using `t`, and then use this in the table.